docs/UserGuide.md

```diff
@@ -43,23 +43,13 @@ You can start from the [Quick Start](#quick-start) section to learn how to obtai
 
 1. Copy the file to the folder you want to use as the _home folder_ for your tCheck.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note
+1. Double-click the file to start the app. The graphic user interface (GUI) similar to the figure below should appear in a few seconds. Note
```

Comment:
Should be Graphical User Interface (capitalise the first letter)

Reply:
Thanks, I think not capitalising would be also accepted.